solvers/revisit_constellation/rgt_apc_gap_constructive/src/solve.py

```
@@ -0,0 +1,334 @@
+            solution_dir / "debug" / "high_gap_intervals.json",
+            envelope_artifacts.high_gap_intervals,
+        )
+        write_json(
+            solution_dir / "debug" / "mode_comparison.json",
+            scheduling_result.mode_comparison,
+        )
+        write_json(
+            solution_dir / "debug" / "adaptation_notes.json",
+            _paper_adaptation_notes(),
+        )
+    except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)